<ui version="4.0" >
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QToolButton" name="open_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Open</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/document-open.png</normaloff>:/img/document-open.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="save_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Save</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/document-save.png</normaloff>:/img/document-save.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="add_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Add</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/list-add.png</normaloff>:/img/list-add.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="erase_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Erase</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/list-remove.png</normaloff>:/img/list-remove.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="up_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Up</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/arrow-up.png</normaloff>:/img/arrow-up.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="down_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Down</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/arrow-down.png</normaloff>:/img/arrow-down.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="clear_button" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Clear</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="../rsclist.qrc" >
          <normaloff>:/img/edit-clear-list.png</normaloff>:/img/edit-clear-list.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="list" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode" >
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode" >
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="shutdown_check" >
      <property name="text" >
       <string>Shutdown when the list finished</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../rsclist.qrc" />
 </resources>
 <connections/>
</ui>
